workflow_ppk.py

```diff
@@ -234,8 +238,11 @@ def main(datadir, geoid, makePos=True, verbose=2, sonar_method='default', rtklib
     elif sonar_method == 'instant':
         sonar_range = sonarData['this_ping_depth_m']
         qualityLogic = sonarData['this_ping_depth_measurement_confidence'] > instant_sonar_confidence
+    elif sonar_method == 'qaqc':
+        sonar_range = sonarData['qaqc_depth_m']
+        qualityLogic = sonarData['qaqc_depth_m'] >= 0
     else:
-        raise ValueError('acceptable sonar methods include ["instant", "smooth"]')
+        raise ValueError('acceptable sonar methods include ["default", "instant", "smooth", "qaqc"]')
```

Comment:
maybe we could make this `["string", "string", "string", "string"]` a variable so you wouldn't have to change a bunch of different places?"

not critical to PR, but maybe a QOL add